beacon-chain/core/state/validity_conditions.go

```diff
@@ -0,0 +1,181 @@
+package state
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/types"
+	v "github.com/prysmaticlabs/prysm/beacon-chain/core/validators"
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	"github.com/prysmaticlabs/prysm/shared/bitutil"
+	"github.com/prysmaticlabs/prysm/shared/hashutil"
+	"github.com/prysmaticlabs/prysm/shared/params"
+)
+
+// IsValidBlock verifies a block is valid according to the ETH 2.0 specification for
```

Comment:
It would be nice to outline the validity conditions here for godocs. Is it an easy copy paste?